#hard_task: "([]<> (r1 && stand)) && ([]<> (r1 && unloaded))"
#soft_task: "[]!r3"
#hard_task: "<>(desk && default && X((carrybin U dispose) && <>default)) && <>r3"
#hard_task: "(<> s2) && (<> s3) && (<> s4) && (<>(s5 && <>s1))"
#hard_task: "<>(s3 && <>(s4 && <>(s2 && <>(s5 && <>s1))))"
#hard_task: "<>s1 && <>s2 && <>s3 && <>s4 && <>s5 && [](s->e) && [](e->!a)"
#hard_task: "<>(r1 && <>(r2 && <>r3)) && <>r4"
#hard_task: "<>r1 && <>r2 && <>r3 && <>r4"
hard_task: "<>((r1 && loaded) U (r3 && X unloaded))"
soft_task: ""